matscipy/dislocation.py

```diff
@@ -478,16 +479,21 @@ def get_elastic_constants(pot_path=None,
     sf = StrainFilter(unit_cell)
     # or UnitCellFilter(W)
     # -> to minimise wrt pos, cell
-    opt = FIRE(sf)
+    if verbose:
+        opt = FIRE(sf)
```

Comment:
@pgrigorev does this have to be FIRE? PreconLBFGS should in general be faster/better behaved.

Reply:
There is no reason for to be `FIRE` happy to switch to `PreconLBFGS`.